Adding documentation for System Nexa 2 Integration (#42586)
home-assistant/home-assistant.github.io: source/_integrations/systemnexa2.markdown

ha_codeowners:
  - '@konsulten'
ha_domain: nexa
ha_dhcp: true
ha_zeroconf: true
ha_quality_scale: bronze
integration_type: device
---

The **System Nexa 2** {% term integration %} allows you to integrate with the **System Nexa 2** devices from [Nexa](https://nexa.se/smarta-hem/system-nexa-2).

## Use cases

The System Nexa 2 integration brings your lights, switches, and smart plugs into Home Assistant, enabling you to create a truly smart home. Here are some ways you can benefit from this integration:

- **Automate your lighting and appliances** - Create automations to turn lights on at sunset, turn off devices when you leave home, or control your car heater based on outdoor temperature and your calendar to ensure a warm car before your morning commute.
- **Enhance privacy and security** - Disable cloud access to keep all communication local, ensuring your devices work without internet connectivity.
- **Reduce light pollution** - Turn off indicator LEDs on devices in your bedroom at night to create a better sleeping environment.
- **Child-proof your smart home** - Disable physical buttons on specific devices to prevent children from turning critical equipment on or off.

> - **Integrate with existing remotes** - Enable 433MHz communication to continue using your existing remote controls alongside Home Assistant automation.

Reviewer: ```suggestion
- Automate your lighting and appliances
   - Create automations to turn lights on at sunset, turn off devices when you leave home, or control your car heater based on outdoor temperature and your calendar to ensure a warm car before your morning commute.
- Enhance privacy and security
   - Disable cloud access to keep all communication local, ensuring your devices work without internet connectivity.
- Reduce light pollution
   - Turn off indicator LEDs on devices in your bedroom at night to create a better sleeping environment.
- Child-proof your smart home
   - Disable physical buttons on specific devices to prevent children from turning critical equipment on or off.
- Integrate with existing remotes
   - Enable 433MHz communication to continue using your existing remote controls alongside Home Assistant automation.
```
We use [bold to markup UI strings](https://developers.home-assistant.io/docs/documenting/general-style-guide). If possible, avoid using it to highlight text.

- WPD-01 Indoor Dimmer Plug

## Prerequisites

1. Open the app store and install the **Nexa Hem** app.
2. Create an account.
3. Add the device by onboarding it to your Wi-Fi, making sure it's on a network reachable from Home Assistant.

{% include integrations/config_flow.md %}

{% configuration_basic %}
Host:
    description: "The IP Address/hostname of your device. You can find it in your **Nexa Hem** app by selecting your device > **Network** > **IP**."
{% endconfiguration_basic %}

## Supported functionality

### Switches

- **Relay**
  - **Description**: Turn on or off switch relay
  - **Available for devices**: WBR-01, WPO-01, WPR-01
- **433Mhz**
  - **Description**: Controls whether device should use 433Mhz communication
  - **Available for devices**: all
- **Cloud Access**
  - **Description**: Controls whether to allow cloud access for device
  - **Available for devices**: all
- **Physical Button**
  - **Description**: Controls whether the onboard button should have any effect on the device
  - **Available for devices**: all
- **LED**
  - **Description**: Controls whether the onboard LED should be lit at any time
  - **Available for devices**: all

## Known limitations

> Currently this integration does not support:

Reviewer: ```suggestion
Currently this integration does not support the following functionality:
```

- Dimming (state, lowest/highest level, dimming method and more)
- Device local scheduling/timers (can however be done via Home Assistant)
- Adding remote control transmitters to control devices
- Setting mode after power loss
- Updating Wi-Fi settings

> - Firmware Upgrade

Reviewer: ```suggestion
- Firmware upgrade
```

> - Local Auth

Reviewer: ```suggestion
- Local authentication
```

---
title: System Nexa 2
description: How to integrate System Nexa 2 devices within Home Assistant.
ha_category:
  - Light
  - Switch
ha_config_flow: true
ha_platforms:
  - light
  - switch
ha_release: 2026.1

> ha_iot_class: Local Push

Reviewer 2: ```suggestion
  - Switch
ha_config_flow: true
ha_platforms:
  - switch
ha_release: 2026.3
ha_iot_class: Local Push
```

ha_codeowners:
  - '@konsulten'
ha_domain: nexa
ha_dhcp: true
ha_zeroconf: true
ha_quality_scale: bronze

> integration_type: device

Reviewer 2: ```suggestion
integration_type: hub
```

---

The **System Nexa 2** {% term integration %} allows you to integrate with the **System Nexa 2** devices from [Nexa](https://nexa.se/smarta-hem/system-nexa-2).

## Use cases

The System Nexa 2 integration brings your lights, switches, and smart plugs into Home Assistant, enabling you to create a truly smart home. Here are some ways you can benefit from this integration:

- Automate your lighting and appliances
   - Create automations to turn lights on at sunset, turn off devices when you leave home, or control your car heater based on outdoor temperature and your calendar to ensure a warm car before your morning commute.
- Enhance privacy and security
   - Disable cloud access to keep all communication local, ensuring your devices work without internet connectivity.
- Reduce light pollution
   - Turn off indicator LEDs on devices in your bedroom at night to create a better sleeping environment.
- Child-proof your smart home
   - Disable physical buttons on specific devices to prevent children from turning critical equipment on or off.
- Integrate with existing remotes

>    - Enable 433MHz communication to continue using your existing remote controls alongside Home Assistant automation.

Reviewer: ```suggestion
   - Enable 433 MHz communication to continue using your existing remote controls alongside Home Assistant automation.
```

---
title: System Nexa 2
description: How to integrate System Nexa 2 devices within Home Assistant.
ha_category:
  - Light
  - Switch
ha_config_flow: true
ha_platforms:
  - light
  - switch
ha_release: 2025.12
ha_iot_class: Local Push
ha_codeowners:
  - '@konsulten'
ha_domain: nexa
ha_dhcp: true
ha_zeroconf: true

> ha_quality_scale: bronze

Reviewer: ```suggestion
ha_quality_scale: bronze
integration_type: device
```

---

The **System Nexa 2** {% term integration %} allows you to integrate with the **System Nexa 2** devices from [Nexa](https://nexa.se/smarta-hem/system-nexa-2).

## Supported devices

The following devices are known to be supported by the integration:

- WBD-01
- WBR-01
- WPD-01
- WPO-01
- WPR-01

## Prerequisites

1. Open the app store and install the **Nexa Hem** app.
2. Create an account.

> 3. Add the device by onboarding it to your wifi, making sure its on a network reachable from Home Assistant.

Reviewer: ```suggestion
3. Add the device by onboarding it to your Wi-Fi, making sure it's on a network reachable from Home Assistant.
```

{% include integrations/config_flow.md %}

> 

Reviewer: can you add a section on the supported functionality?

---
title: System Nexa 2
description: How to integrate System Nexa 2 devices within Home Assistant.
ha_category:
  - Light
  - Switch
ha_config_flow: true
ha_platforms:
  - light
  - switch

> ha_release: 2025.12

Reviewer: ```suggestion
ha_release: 2026.1
```
update to next possible release

---
title: System Nexa 2
description: How to integrate System Nexa 2 devices within Home Assistant.
ha_category:
  - Light
  - Switch
ha_config_flow: true
ha_platforms:
  - light
  - switch
ha_release: 2026.1
ha_iot_class: Local Push
ha_codeowners:
  - '@konsulten'
ha_domain: nexa
ha_dhcp: true
ha_zeroconf: true
ha_quality_scale: bronze
integration_type: device
---

The **System Nexa 2** {% term integration %} allows you to integrate with the **System Nexa 2** devices from [Nexa](https://nexa.se/smarta-hem/system-nexa-2).

## Supported devices

The following devices are known to be supported by the integration:

- WBD-01
- WBR-01
- WPD-01
- WPO-01
- WPR-01

## Prerequisites

1. Open the app store and install the **Nexa Hem** app.
2. Create an account.
3. Add the device by onboarding it to your Wi-Fi, making sure it's on a network reachable from Home Assistant.

{% include integrations/config_flow.md %}

## Supported functionality

> ### Control

Reviewer: normally, the subheaders usesd in the supported functionality reflect the type. Switch, sensor, light, etc.
For example:
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L88

Author: Updated the section please check :)

> #### Control

Reviewer: ```suggestion
## Supported functionality

### Control
```

- On
- Off

> #### Configuration

Reviewer: ```suggestion
### Switches
```

Author: Should be updated now =)
